source/common/upstream/maglev_lb.h

```diff
@@ -14,8 +14,8 @@ namespace Upstream {
  * All Maglev load balancer stats. @see stats_macros.h
  */
 #define ALL_MAGLEV_LOAD_BALANCER_STATS(GAUGE)                                                      \
-  GAUGE(max_entries_per_host, Accumulate)                                                          \
-  GAUGE(min_entries_per_host, Accumulate)
+  GAUGE(max_entries_per_host, NeverImport)                                                         \
+  GAUGE(min_entries_per_host, NeverImport)
```

Comment:
This is not a comment about you, and more about the complexity of this entire situation, but I think we have a 0% success rate with landing changes like this without a regression. I think once we decide on the approach can we land the Accumulate -> NeverImport changes in an independent PR?